charts/substrate-faucet/values.yaml

```
@@ -27,11 +27,11 @@ bot:
     repository: paritytech/faucet-bot
     tag: latest
     pullPolicy: Always
-
+  existingConfigMap: ''
+  existingSecret: ''
```

```suggestion
  existingConfigMap: ""
  existingSecret: ""
```
use double quotes for consistency?